changes/views.py

```diff
@@ -287,6 +287,24 @@ def post(self, request, *args, **kwargs):
         return Response(status=status.HTTP_202_ACCEPTED)
 
 
+class ReceiveWhatsAppTimeoutSystemEvent(ReceiveWhatsAppBase):
+    serializer_class = ReceiveWhatsAppSystemEventSerializer
+
+    def post(self, request, *args, **kwargs):
+        self.validate_signature(request)
+        serializer = self.get_serializer(data=request.data)
+        if not serializer.is_valid():
+            return Response(status=status.HTTP_400_BAD_REQUEST)
+
+        for item in serializer.validated_data["statuses"]:
+            for error in serializer.validated_data["errors"]:
+                tasks.process_whatsapp_timeout_system_event.delay(
+                    item["id"], item["timestamp"], item["recipient_id"], item["errors"]
+                )
+
+        return Response(status=status.HTTP_202_ACCEPTED)
```

Comment:
I think we rather want to expand the existing view https://github.com/praekeltfoundation/ndoh-hub/blob/ee891d316964398f834b725ec84242d212dc6cdc/changes/views.py#L239-L272 instead of making a new view to handle this. The events are already going to that old view, it's just that nothing is being done with them

Reply:
when we last talked, you said to create a new view, and if the new one passes, we delete the old one

Reply:
Sorry, I think we misunderstood each other. I meant that we add this functionality to the existing `ReceiveWhatsAppEvent` view, and then we can delete the old `ReceiveWhatsAppSystemEvent` view